ux-guidelines.md

```diff
@@ -0,0 +1,177 @@
+# Secure Payment Confirmation: Non-normative UX Guidelines
+
+Status: This is a **draft** document without consensus.
+
+A key step of the user journey for Secure Payment Confirmation (SPC) is the
+display of and user interaction with the [transaction confirmation UX]. This
+step communicates information from the caller of SPC (e.g., a merchant website,
+perhaps on behalf of a payment service provider or issuing bank) to the user,
+for the user to verify and ultimately authenticate via WebAuthn.
+
+While web specifications do not generally make recommendations about how User
+Agents implement UX, both integrators of SPC (e.g., financial organizations)
+and implementors of SPC (i.e., User Agents) have requested general guidance
+about what information can be provided to the API and how it is likely to be
+displayed to the user. The non-normative guidance in this document is based on
+implementor experience and payment industry feedback.
```

Comment:
nit: is this important context for the audience reading this, or just us? Might be possible to compact this down to a single sentence like: `While web specifications do not generally make recommendations about how User Agents implement UX, we wanted to offer general guidance about how the API's information will likely be displayed to the user.` or just remove it entirely.

Feel free to ignore my nits if you are confidant 😄